### ISSUE TITLE
Innovation and Planning Sprint (FY25 Q1)

### ISSUE THREAD

**Issue by nathankota** [agrc/planning-queue#447]
### Benefit

The final sprint of the quarter is meant to be an [Innovation and Planning (IP) Iteration](https://scaledagileframework.com/innovation-and-planning-iteration/). Benefits include:
- Allow time for Innovation
- Dedicate time to PI Events (I&A and PI planning)
- Integrate the Complete Solution (presentation of work of all teams on the train)
- Advance Development Infrastructure
- Enable Continuous Learning

Possible tasks include:
- Pay down technical debt
  - app and data dependencies and updates
  - updating CIs
- Other modernization tasks
- Cloud migration
- "Work on whatever with whomever"
- Alignment of best practices

#### Possible Tasks 

- [ ] https://github.com/agrc/darwin/discussions/14 (scheduled for 9/16)
- [ ] https://github.com/agrc/porter/issues/324
- [ ] Remove [reactfire](https://github.com/FirebaseExtended/reactfire/pull/594)
   - [ ] https://github.com/agrc/deq-enviro
   - [x] https://github.com/agrc/atlas
   - [ ] https://github.com/agrc/api.mapserv.utah.gov
   - [ ] https://github.com/agrc/plss
   - [ ] https://github.com/agrc/turn-gps-billpay 
- [ ] Palletjack: https://github.com/agrc/planning-queue/issues/274
- [ ] Update Skid readme's
  - [ ] https://github.com/agrc/uorg-skid/issues/3
  - [ ] https://github.com/agrc/dabs-skid/issues/4
  - [x] https://github.com/agrc/wmrc-skid/issues/33
  - [ ] https://github.com/agrc/broadband-speedtest-skid/issues/5
- [ ] Release pipeline improvements
   - Separate deployments releases by moving them to a separate workflow that is triggered by published releases. 
   - This would make it easier to rerun them if there was an issue with the deploy.
- [x] Migrate apps to print proxy v2
- [ ] Migrate functions to 2nd generation 
   - [Gen 1 function list](https://docs.google.com/spreadsheets/d/1oYFvYKzubULZnJsAePofCi-L5uYWleN7OUvSoQrZIyo/edit?gid=0#gid=0)
   - plan and approvals for public functions RITM0407680
- [ ] Implement https://github.com/agrc/release-issue-notifications-action
- [ ] https://github.com/agrc/nfhl-skid/issues/25
- [ ] Consider https://github.com/step-security/harden-runner
- [ ] https://github.com/agrc/skid/issues/23
- [ ] Convert skid template to cloud run instead of function
- [x] GitHub audit
- [ ] Cloud project elcids (including our own as it has changed)

API Enhancements:
- [ ] https://github.com/agrc/api.mapserv.utah.gov/issues/117

### Issue Reference

refs #376

**Comment by stdavis** [comment]
I checked off the print proxy migration. The only app left on [the list](https://github.com/agrc/serverless-print-proxy/issues/205) is WRI which will get folded into the rewrite, I assume.

**Comment by steveoh** [comment]
### Sprint 6 Notes

- Built a rudimentary GitHub org audit 
   - The results are stored in BigQuery for analysis
   - A Looker studio project was created with some basic charts and graphs
- Terraforming GitHub projects has started with a WIP for javascript to the cloud projects
- We removed reactfire from atlas and moved the providers and code to the utah-design-system package for reuse
- We started the AGOL backup and restore system. Backups will start running soon and we can prioritize working on the website and restore in the future.
- We've picked away at separating the release pipeline from the deploy but have more projects to do before we can complete that

**Comment by steveoh** [comment]
@stdavis and @jacobdadams will you update this and then we can close it out?

**Comment by stdavis** [comment]
Most of my time was spent on https://github.com/agrc/project-moonwalk.
Also, tore out my release-please fork from our release composite action. This should speed up this step in our workflows.

**Comment by jacobdadams** [comment]
I wasn't able to contribute to any of the listed items in a meaningful way this sprint, with the possible exception of the initial moonwalk discussion.
